src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/DRConfig/SetAzureServiceBusDRConfigurationBreakPairing.cs

```diff
@@ -0,0 +1,58 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ServiceBus.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.ServiceBus.Commands.GeoDR
+{
+    /// <summary>
+    /// 'New-AzureRmServicebusDRConfigurationsBreakPairing' Cmdlet disables the Disaster Recovery and stops replicating changes from primary to secondary namespaces
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, ServicebusDRConfigurationBreakPairingVerb)]
```

Comment:
@v-Ajnava this cmdlet needs an output type and to support `ShouldProcess`

Reply:
implemented